spec/request/vet360/telephone_request_spec.rb

```diff
@@ -155,5 +160,78 @@
         expect(errors_for(response)).to include "phone-number - can't be blank"
       end
     end
+
+
+
+
+
+    context 'when effective_end_date is included' do
+
+      let(:telephone) { 
+        build(:telephone,
+          vet360_id: user.vet360_id, 
+          effective_end_date: Time.now.utc.iso8601,
+          phone_number: '5551234'
+        ) 
+      }
+
+      before do
+        allow_any_instance_of(User).to receive(:icn).and_return('1234')
+        telephone.id = 1299
+      end
+
+
+      it 'effective_end_date is ignored', {:aggregate_failures => true} do
+        VCR.use_cassette('vet360/contact_information/put_telephone_ignore_eed', VCR::MATCH_EVERYTHING) do
+
+          put(
+            '/v0/profile/telephones',
+            telephone.to_json,
+            auth_header.update(
+              'Content-Type' => 'application/json', 'Accept' => 'application/json'
+            )
+          )
+
+          expect(response).to have_http_status(:ok)
+          expect(response).to match_response_schema('vet360/transaction_response')
+        end
+      end
+    end
+
+
+
+
+
+  end
+
+  describe 'DELETE /v0/profile/telephones' do
+    let(:telephone) { 
+      build(:telephone, vet360_id: user.vet360_id) 
+    }
+
+    before do
+      allow_any_instance_of(User).to receive(:icn).and_return('64762895576664260')
+      telephone.id = 42
+    end
+
+    context 'with a 200 response' do
+      it 'should match the telephone schema', :aggregate_failures do
+        VCR.use_cassette('vet360/contact_information/delete_telephone_success', VCR::MATCH_EVERYTHING) do
+
+          delete(
+            '/v0/profile/telephones',
+            telephone.to_json,
+            auth_header.update(
+              'Content-Type' => 'application/json', 'Accept' => 'application/json'
+            )
+          )
+
+          expect(response).to have_http_status(:ok)
+          expect(response).to match_response_schema('vet360/transaction_response')
+        end
+      end
+    end
   end
 end
+
```

Comment:
I haven't been following the delete stuff as closely as you two have.  That said, I am not seeing how this is testing that something was/was not deleted.  I see that it is testing that a successful transaction was processed.  Any way to add in testing to confirm that a field was there before, and then was ultimately deleted?